### DISCUSSION TITLE
Add a way for users to add (additional) phonenumbers to their account

### DISCUSSION THREAD

**Discussion by olmari** [TampereHacklab/mulysa#383] [Ideas]
Like for NFC in #150 make it possible for users to add more phonenumbers to their account. As users can have more phonenumbers than one which they want to use for door access. Thus phonenumbers in door access point needs to be thought more of an "token" just like NFC cards would.

Asylum has separately an phonenumer for contact and access token phonenumber. New member applying monthly fee membership, the supplied phonenumber gets automatically copied into an token on approval, and more "phone tokens" can be manually added later. I think something alike would be ultimately nice approach in long run.

**Comment by vranki** [comment]
How do we prevent members from adding other person's phone numbers to their account? I think this should not be a self managed thing if implemented.

**Comment by olmari** [comment]
We don't generally survey members access tokens, we need to trust our members in general standpoint. Nothing stops an member from adding any arbitary phonenumber to their account, but why would average user risk own access or membership at the worst either.. Wrongdoings need to be dealt within board meeting when such thing arise anyway and this doesn't differ. Approval process per default would just unneccesarily generate more burden to "staff" for no reason.

One idea: In technical standpoint, maybe there could exist an setting per "token type" that dictates does add/change go through approval process, should there inevidetably be diffirent needs and workload models between Mulysa users.
